site/src/request_handlers/next_artifact.rs

```diff
@@ -1,9 +1,22 @@
 use crate::load::{MissingReason, SiteCtxt};
-use collector::api::next_commit;
+use collector::api::next_artifact;
 
 use std::sync::Arc;
 
-pub async fn handle_next_commit(ctxt: Arc<SiteCtxt>) -> next_commit::Response {
+pub async fn handle_next_artifact(ctxt: Arc<SiteCtxt>) -> next_artifact::Response {
+    // Prefer benchmarking released artifacts first
+    match ctxt.missing_artifacts().await {
+        Ok(next_artifact) => {
+            if let Some(next_artifact) = next_artifact.into_iter().next() {
+                log::debug!("next_artifact: {next_artifact}");
```

Comment:
This debug statement doesn't really mean much outside the context of the code. perhaps we can add a bit more context to this?

Reply:
Well, it was inspired by a similar comment in the same function :sweat_smile::
```rust
log::debug!(
            "next_commit: {} (missing: {})",
            commit.sha,
            missing_reason_dbg
        );
```
What additional context should be included?